blocks/carousel/carousel.js

```diff
@@ -0,0 +1,49 @@
+import { button, div, span } from '../../scripts/dom-helpers.js';
+import { decorateIcons, loadCSS } from '../../scripts/lib-franklin.js';
+import { decorateAPICards, decorateCuratedCards } from '../cards/cards.js';
+
```

Comment:
Make the carousel draggable. See example: https://github.com/hlxsites/zemax/blob/d99551b256dedf765f6de4cc1f4162963f938377/blocks/cards-carousel/cards-carousel.js#L185